cypress/support/commands.js

```diff
@@ -4,7 +4,12 @@ import { Constants } from '../constants/constants'
 const constants = new Constants();
 
 require('cy-verify-downloads').addCustomCommand();
-Cypress.Commands.add('login', (username = Cypress.env('username'), password = Cypress.env('password')) => {
+
+Cypress.Commands.add('login', (params = {}) => {
+    const url = params.url || constants.dashboardUrl;
+    const username = params.username ||   Cypress.env('username');
```

Comment:
1. Create Vlan2   (As expected, because our environment vlan2 does not support it)

2. case2 & case 3,   （Resolved, image name needs to be converted to lower case）